frontend/src/metabase-lib/v1/expressions/formatter/formatter.ts

```diff
@@ -54,10 +55,22 @@ export async function formatExpressionParts(
   root: Lib.ExpressionParts | Lib.ExpressionArg,
   options: FormatOptions = {},
 ) {
+  let prettierFormat: typeof staticPrettierFormat;
+
+  if (!process.env.IS_EMBEDDING_SDK) {
```

Comment:
nit: shall we import from `import { isEmbeddingSdk } from "metabase/env"` instead? makes it easier to keep track of where we reference this environment variable.

Reply:
No, we have to explicitly use process.env so bundler can remove dead code